lighthouse-core/report/handlebar-helpers.js

```diff
@@ -161,7 +195,7 @@ const handlebarHelpers = {
    * @param {!Object} opts
    * @return {string}
    */
-  createTreeRenderContext(tree, opts) {
+  createTreeRenderContext(tree = {}, opts) {
```

Comment:
this was to work around a wonky unit test in development, but no longer necessary \o/ removed.